content/docs/analytics/Implement.md

```diff
@@ -188,3 +188,33 @@ Don't worry! `ReactiveSearch` handles this for you. You just need to set the `an
 
 - When a user types something in sequence, for example: `b→bo→boo→book`, we understand the context and instead of creating different search sessions, we count it as a search session with search key stored as `book`.
 - By default, the search context is valid for up to `30s`. For example, `b→bo→boo...30 seconds...->book` will be recorded as two different searches with keys as `boo` & `book`.
+
+## How Do We Record A User Session?
+- A user session starts when someone hits the Appbase.io search request from any platform.
+- A user session automatically ends after `30` minutes but if someone is continuously interacting with the search then we change the end duration to the next `30` minutes from the last interaction. 
+
+For example:
+
+=> Bob made the first request at `10:00` then the session will end at `10:30`,
+
+=> If Bob has made some click or searched for a new term at `10:05` then the session will be extended till `10:35`.
+
+- We keep track of user sessions with the help of `IP` address. However, if you're using the `User ID` in the `search` request then it'll be used instead of `IP`. So, a change in the `User ID` in the search request will start a new session.
+
+
+## What Is A Bounce For Appbase.io Search Users?
+
+> Bounce represents the percentage of visitors who enter the site and then leave ("bounce") rather than continuing to view other pages within the same site.
+
+That's what [wikipedia](https://en.wikipedia.org/wiki/Bounce_rate) says about `Bounce Rate` but if we talk about it in the context of appbase.io which is all about `search` then a `Bounce` user can be defined as the `user` who visited your search page (the page which makes at least one search request to the appbase.io) and then leave without doing any further interactions with **search**.
```

Comment:
This can be phrased better. I will do it if it's not obv.

Just define how we calculate bounce rate (we don't need to explain the distinction to start with), can explain it after we define it if needed.